lib/matplotlib/axes/_base.py

```diff
@@ -985,14 +1040,30 @@ def clear(self):
         """clear the axes"""
         self.cla()
 
+    def set_prop_cycle(self, prop_cycle):
+        """
+        Set the prop cycle for any future plot commands on this Axes.
+
+        *prop_cycle* is a :class:Cycler object.
+        Can also be `None` to reset to the cycle defined by the
+        current style.
+        """
+        self._get_lines.set_prop_cycle(prop_cycle)
+        self._get_patches_for_fill.set_prop_cycle(prop_cycle)
+
     def set_color_cycle(self, clist):
         """
         Set the color cycle for any future plot commands on this Axes.
 
         *clist* is a list of mpl color specifiers.
+
+        .. deprecated:: 1.5
```

Comment:
Then I suggest we deprecate this method, otherwise it feels inconsistent with the other styles... I would only want to keep it if we had this and similar methods as shortcuts to add/replace parts of the property cycle...